setup.cfg

```diff
@@ -38,6 +38,9 @@ artifact_indexes=
     # https://ci.appveyor.com/project/sklearn-ci/scikit-learn/
     http://windows-wheels.scikit-learn.org/
 
+[flake8]
+ignore=E121,E123,E126,E24,E704,W503,W504
```

Comment:
Are they actually needed in the sense that they should correspond to the default ignore flake8 settings if I believe the comment in master?

```bash
# Default ignore PEP8 violations are from flake8 3.3.0
DEFAULT_IGNORED_PEP8=E121,E123,E126,E226,E24,E704,W503,W504
```

Reply:
From the issue that lead me to do this pr, I think these are truly project wide settings in that they are not something that the maintainers feel are worth worrying about.

Reply:
@lesteve the default ignored list can change from version to version so its better to be explicit. Additionally, the current default for the most recent version ignores e226 which is something worth enforcing according to the issue this pr was in response to.

Reply:
I am not 100% convinced, I would rather trust that flake8 default is a good default. As an example: it could well be that flake8 add a completely new kind of violations that they don't want to be shown by default. Setting `ignore` explicitly is not the best thing to do in this case.